InfiniteLoopBookProject/src/com/st/lms/service/Service.java

```diff
@@ -0,0 +1,208 @@
+package com.st.lms.service;
+
+import com.st.lms.model.*;
+import com.st.lms.dao.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Service {
+	private static DAOBookImpl bookdao = new DAOBookImpl();
+	private static DAOAuthorImpl authordao = new DAOAuthorImpl();
+	private static DAOPublisherImpl publisherdao = new DAOPublisherImpl();
+	
+	//cmds to manipulate book table
+	public static void printBooks() {
+		List<Book> books = new ArrayList<Book>();
+		DAOBookImpl bd = new DAOBookImpl();
+		try {
+			books = bd.loadBooks();
+			System.out.println(String.format("%-13s %-5s %-30s %-5s %-13s %-5s %-15s", "BookId", "|", "Title", "|", "AuthorId", "|", "PublisherId"));

```

Comment:
Instead of `System.out.println(String.format(...))`, just use `System.out.printf(...)`.